crates/subnetworks/beacon/src/storage.rs

```diff
@@ -861,4 +899,34 @@ mod test {
         let result = storage.get(&key).unwrap().unwrap();
         assert_eq!(result, value.as_ssz_bytes());
     }
+
+    #[test]
+    fn test_beacon_storage_prune_old_bootstrap_data() {
+        let (_temp_dir, config) = create_test_portal_storage_config_with_capacity(10).unwrap();
+        let mut storage = BeaconStorage::new(config).unwrap();
+        let mut value = test_utils::get_light_client_bootstrap(0);
+        // Set the bootstrap slot to outdated value
+        let current_slot = expected_current_slot();
+        let outdated_slot = current_slot - (4 * 30 * 24 * 60 * 60 / 12) - 1;
```

Comment:
```suggestion
        let outdated_slot = current_slot - (33042 * 32) - 1;
```

due to the reasoning above